src/main/java/gui/controllers/RoomController.java

```diff
@@ -73,7 +73,10 @@ private void newItemDone(final int index) {
         progress.updateProgress();
         int amountNewChests = progress.getRoom().getChestsOpened() - chestsOpened;
         for (int i = 0; i < amountNewChests; i++) {
-            cameraHandler.getInformationHandler().addInformation("Found chest");
+            int chestNumber = i + chestsOpened + 1;
+            String chestsFound = chestNumber + "/"
+                + progress.getRoom().getChestList().size();
+            cameraHandler.getInformationHandler().addInformation("Found chest " + chestsFound);
         }
```

Comment:
is het misschien handig om te laten zien dat er een kist minder is als er in de progress bar een kist wordt verwijderd